pkg/storegateway/bucket_stores.go

```diff
@@ -180,6 +180,14 @@ func (u *BucketStores) Series(req *storepb.SeriesRequest, srv storepb.Store_Seri
 	return store.Series(req, srv)
 }
 
+func (u *BucketStores) HasUser(userID string) bool {
+	u.storesMu.RLock()
+	defer u.storesMu.RUnlock()
+
+	_, ok := u.stores[userID]
+	return ok
+}
+
```

Comment:
Why exported? (Also, why is it needed? `getStore(...) != nil` works too)

Reply:
Very good point. I introduced `HasUser()` when it still was in the `pkg/querier` and now it's not more required: `getStore()` is enough.